adagrams/game.py

```diff
@@ -1,11 +1,105 @@
+import random
+
+
+
 def draw_letters():
-    pass
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    drawn_letters = [] 
+    for letter in range(10):
+        my_current_key = random.choice(list(LETTER_POOL))
+        my_current_value = LETTER_POOL.get(my_current_key)
+        if my_current_value != 0:
+            drawn_letters.append(my_current_key)
+            LETTER_POOL[my_current_key] = my_current_value -1
+        if LETTER_POOL[my_current_key] == 0:
+            LETTER_POOL.pop(my_current_key)
+    return drawn_letters
+            
```

Comment:
Overall this function looks good, but it doesn't fully address the weighted nature of the alphabet's randomness. Your random choice picks a letter randomly but each letter has the same weight. Is there a way you could extend that dictionary into a list that holds every letter the correct number of times it appears?